docs/pages/guides/configuration-management/importing-applications.md

```diff
@@ -50,7 +52,7 @@ This command enable users to import their existing applications from sources as
 
 **Step 1: Access the Meshery UI**
 
-To import a Docker Compose app into Meshery using the Meshery UI, you must first [install Meshery](../installation/quick-start.md)
+To import a Docker Compose app into Meshery using the Meshery UI, you must first [install Meshery](/installation/quick-start)
```

Comment:
Why are there changes here, this isn't part of the issue description.

Reply:
Those are commits from the issue #9991

Reply:
Thanks for this review, @sandramsc 👏

Reply:
@neerajmanivarnan, don't include commits from a separate PR into another PR.